src/StaticLint.jl

```diff
@@ -188,7 +188,7 @@ function followinclude(x, state::State)
             state(getcst(state.file))
             state.file = oldfile
             pop!(state.included_files)
-        elseif !is_in_fexpr(x, CSTParser.defines_function)
+        elseif !is_in_fexpr(x, CSTParser.defines_function) && !isempty(init_path)
```

Comment:
I this pass, `init_path` is actually always not empty, no ? Because there is `isempty(path)` escape in the first condition pass.

Reply:
No, this is a new `if` block starting [here](https://github.com/julia-vscode/StaticLint.jl/pull/211/files#diff-12cbe5742f403d010ed4f79810d15e1eR178).

Reply:
ah I missed that. LGTM.